DEVELOPER_ONBOARDING.md

```
@@ -0,0 +1,362 @@
+WandererKills is a real-time EVE Online killmail data service that:
+- Streams killmail data from zKillboard
+- Enriches data with EVE Swagger Interface (ESI) 
+- Provides REST, WebSocket, and Server-Sent Events APIs
+- Handles 10,000+ concurrent connections
+- Processes 1,000+ killmails per minute
+
+## Quick Start
+
+```bash
+# Install dependencies
+mix deps.get
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)